### ISSUE TITLE
[BUG] Better Rolls: Attack roll Token number only visible to the player to who rolled it

### ISSUE THREAD

**Issue by DM-miX** [arcanistzed/scs#8]
**Describe the bug**
When rolling attacks, As the GM I cannot see the number rolled by other players.
Players can see the number on their token for their roll

**To Reproduce**
Roll Attacks in the Attack phase

**Foundry VTT version:**
0.8.9

**Comment by arcanistzed** [comment]
Are you using the Better Rolls module? If you are I would recommend taking a look at the notes in the README: https://github.com/arcanistzed/scs/blob/main/README.md

**Comment by arcanistzed** [comment]
Can you see the rolls in the chat tab?

**Comment by DM-miX** [comment]
Yep using Better Rolls. I had a read, that doesn't seem to be the problem.

The HUD appears on our tokens, but we can only see the rolls we made ourselves.
ie. If a player rolls, they can see the HUD, but noone else can (including me as DM). If I roll for that player I can see the HUD but they can't.

**Comment by arcanistzed** [comment]
Do you have MidiQoL or another rolling module enabled, maybe?

**Comment by DM-miX** [comment]
Nup - no other rollers... wait - Dice So Nice if that counts

**Comment by DM-miX** [comment]
I'll run some test with find the culprit and see what I come up with

**Comment by DM-miX** [comment]
With only SCS (+intro) and Better Rolls (+libwrapper) enabled the issue still persists.
Also noticed that if I do a default sheet roll from the character sheet that the HUD doesn't appear at all.

**Comment by arcanistzed** [comment]
Does it work if you select the token first?

**Comment by DM-miX** [comment]
yeah sure did

**Comment by DM-miX** [comment]
That's an ambiguos answer - yes we are selecting the token first. No it still doesn't show for anyone else

**Comment by arcanistzed** [comment]
Sorry for the long wait. I haven't been able to reproduce this issue at all, so I can't fix it. I could join your game to see the issue myself if you'd like. If not, I'll have to close the issue 🙂

**Comment by arcanistzed** [comment]
> I Roll an attack using Better Rolls. The hud token appears on the players screen, but not the DM’s. Vice versa when I roll for a player on the DM screen, the player cannot see the hud token.

> I discovered that if automatic rolling of damage is disabled and the DM clicks the button to roll damage, then they get to see the attack roll above player's tokens. But that's obviously not ideal.
> However, this is likely just related to the original problem of the DM only being able to see icons for tokens that they personally roll for.
